django-stubs/contrib/admin/options.pyi

```diff
@@ -76,19 +76,19 @@ _ListFilterT: TypeAlias = (
 _ModelT = TypeVar("_ModelT", bound=Model)
 
 class BaseModelAdmin(Generic[_ModelT]):
-    autocomplete_fields: Sequence[str]
-    raw_id_fields: Sequence[str]
+    autocomplete_fields: _ListOrTuple[str]
+    raw_id_fields: _ListOrTuple[str]
     fields: _FieldGroups | None
-    exclude: Sequence[str] | None
+    exclude: _ListOrTuple[str] | None
     fieldsets: _FieldsetSpec | None
     form: type[forms.ModelForm[_ModelT]]
-    filter_vertical: Sequence[str]
-    filter_horizontal: Sequence[str]
+    filter_vertical: _ListOrTuple[str]
+    filter_horizontal: _ListOrTuple[str]
     radio_fields: Mapping[str, _Direction]
     prepopulated_fields: dict[str, Sequence[str]]
     formfield_overrides: Mapping[type[Field], Mapping[str, Any]]
-    readonly_fields: Sequence[str]
-    ordering: Sequence[str] | None
+    readonly_fields: _ListOrTuple[str] | None
```

Comment:
I don't think `readonly_fields` is allowed to be `None`?

Reply:
Oh, nice find. This was a copy paste mistake..